bugbug/models/bugtype.py

```diff
@@ -27,6 +27,7 @@
     "crash": "crash",
     "crashreportid": "crash",
     "perf": "performance",
+    "rca": "rca",
```

Comment:
"rca" is not a type, it's just a way to specify what the type is (e.g. `[rca - Unhandled exceptions]`).
You can find a few example bugs with this query: https://bugzilla.mozilla.org/buglist.cgi?status_whiteboard_type=allwordssubstr&status_whiteboard=%5Brca&query_format=advanced&resolution=FIXED.

Reply:
Ok, I see. If I have a bug with whiteboard `[rca - XYZ]`, how should I infer which category XYZ comes in (from the list of bug types we have at the moment).
Is there a list somewhere?

Reply:
For now, consider XYZ to be the category. Then we can see which categories we have and check if we can merge any of them with the existing ones.